python/lsst/meas/astrom/astrometryNetDataConfig.py

```diff
@@ -3,6 +3,9 @@
 mechanism, but we need nested lists, so we do this home-brew version
 instead.
 '''
+from past.builtins import execfile
```

Comment:
Fix what? I would have thought all the past and builtins imports would be fixed together when we go to python3 only?